 --- app/code/core/Mage/Checkout/controllers/OnepageController.php	(revision )
 +++ app/code/core/Mage/Checkout/controllers/OnepageController.php	(revision )
 @@ -476,7 +476,7 @@

              // get section and redirect data
              $redirectUrl = $this->getOnepage()->getQuote()->getPayment()->getCheckoutRedirectUrl();
 -            if (empty($result['error']) && !$redirectUrl) {
 +            if (empty($result['error'])) {
                  $this->loadLayout('checkout_onepage_review');
                  $result['goto_section'] = 'review';
                  $result['update_section'] = array(

--- app/code/local/Icommerce/CheckoutCommon/Model/Checkout/Onepage.php	(revision )
+++ app/code/local/Icommerce/CheckoutCommon/Model/Checkout/Onepage.php	(revision )
@@ -1975,6 +1975,12 @@
                 break;
         }

+        if ($this->getQuote()->getPayment()->getMethod() == Mage_Paypal_Model_Config::METHOD_WPP_EXPRESS) {
+            $this->getCheckout()->setRedirectUrl($this->getQuote()->getPayment()->getCheckoutRedirectUrl());
+
+            return;
+        }
+
 //        $this->LogIfGuest("C",NULL);

         $service = Mage::getModel('sales/service_quote', $this->getQuote());
